docs/en/ingest-management/elastic-agent/install-elastic-agent.asciidoc

```diff
@@ -51,3 +51,14 @@ Refer to:
 * {eck-ref}/k8s-elastic-agent.html[Run {agent} on ECK] -- for {eck} users
 --
 
+[discrete]
+== Minimum Requirements
+
+Minimum requirements have been determined by running the {agent} (`v8.0.0`) on a GCP `e2-micro` instance (2vCPU/1GB).
+The {agent} used the default policy, running the system integration and self-monitoring.
+
++ **CPU** Under 2% total, including all monitoring processes
++ **Disk** 640MB
++ **RSS Mem Size** 300 MB
+
```

Comment:
```suggestion
|===
| **CPU** | Under 2% total, including all monitoring processes
| **Disk** | 640MB
| **RSS Mem Size** | 300 MB
|===
```

Reply:
The formatting here looks wrong in the output. Sorry I didn't notice this before! I've added markup for a table, but you could format this as a list instead.